KarhooSDK/Service/Auth/LoginInteractor/KarhooAuthLoginWithTokenInteractor.swift

```diff
@@ -12,26 +12,31 @@ final class KarhooAuthLoginWithTokenInteractor: AuthLoginWithTokenInteractor {
     private var token: String?
     private let tokenExchangeRequestSender: RequestSender
     private let userInfoSender: RequestSender
+    private let nonceRequestSender: RequestSender
+    private let paymentProviderRequest: RequestSender
     private let userDataStore: UserDataStore
     private let analytics: AnalyticsService
-    private let paymentProviderRequest: RequestSender
-    private let loyaltyProviderRequest: RequestSender
-    private let nonceRequestSender: RequestSender
+    private let paymentProviderUpdateHandler: PaymentProviderUpdateHandler
 
     init(tokenExchangeRequestSender: RequestSender = KarhooRequestSender(httpClient: JsonHttpClient.shared),
          userInfoSender: RequestSender = KarhooRequestSender(httpClient: TokenRefreshingHttpClient.shared),
+         paymentProviderRequest: RequestSender = KarhooRequestSender(httpClient: JsonHttpClient.shared),
+         nonceRequestSender: RequestSender = KarhooRequestSender(httpClient: TokenRefreshingHttpClient.shared),
          userDataStore: UserDataStore = DefaultUserDataStore(),
          analytics: AnalyticsService = KarhooAnalyticsService(),
-         paymentProviderRequest: RequestSender = KarhooRequestSender(httpClient: JsonHttpClient.shared),
-         loyaltyProviderRequest: RequestSender = KarhooRequestSender(httpClient: JsonHttpClient.shared),
-         nonceRequestSender: RequestSender = KarhooRequestSender(httpClient: TokenRefreshingHttpClient.shared)) {
+         paymentProviderUpdateHandler: PaymentProviderUpdateHandler? = nil
```

Comment:
If default value is added anyway, we don't add default value to constructor right away? The same goes for next examples.

Reply:
Because we cannot use parameter from init method as a default, "Cannot use instance member 'nonceRequestSender' as a default parameter"